src/bin/pgcopydb/summary.c

```
@@ -434,7 +434,7 @@ summary_add_table(DatabaseCatalog *catalog, CopyTableDataSpec *tableSpecs)
 	}
 
 	char *sql =
-		"insert into summary(pid, tableoid, partnum, start_time_epoch, command)"
+		"insert or replace into summary(pid, tableoid, partnum, start_time_epoch, command)"
```

I'm wondering if that could case issues wrt hiding errors.